src/docGenerator.ts

```diff
@@ -17,6 +17,7 @@ import { toolName, toolVersion } from "./toolMetadata"
 import { writeFile } from "./fileUtils"
 import { capitalize, patternTitle } from "./docGeneratorStringUtils"
 import { Rule } from "eslint"
```

Comment:
I would put external dependency import before relative dependencies

Reply:
Added eslint plugin to sort imports 👌